runtime/src/lib.rs

```diff
@@ -1173,8 +1151,8 @@ construct_runtime!(
         // Verifiers. Start indices at 160 to leave room and to the end (255). Don't add
         // any kind of other pallets after this value.
         CommonVerifiers: pallet_verifiers::common = 160,
-        SettlementFFlonkPallet: pallet_fflonk_verifier = 161,
-        SettlementZksyncPallet: pallet_zksync_verifier = 162,
+        // SettlementFFlonkPallet: pallet_fflonk_verifier = 161,
+        // SettlementZksyncPallet: pallet_zksync_verifier = 162,
```

Comment:
please remove the commented code. Moreover if this PR will be included in the regenesis release we can also push up the other verifiers indexes to close the hole. Ok, I thought about that: we can close the gap in any case because the relay chain is not released yet in any place